documentation/element-api/tutorial-properties-attributes.asciidoc

```diff
@@ -115,18 +143,19 @@ element.getStyle().remove("background-color");
 element.getStyle().has("cursor");
 ----
 
-[NOTE]
-You cannot set the element's _style_ attribute using the `element.setAttribute` method.
-However you can use `element.getAttribute('style')` to get all the style rules in a single concatenated string.
+Note:
 
-[[textContent]]
-== TextContent
+* You cannot set an element's `style` attribute using the `element.setAttribute` method. 
```

Comment:
Let's remove this bullet, that works just fine, although it is easier to use the getStyle helper.

Reply:
removed